integration-cli/docker_cli_volume_test.go

```diff
@@ -20,6 +20,16 @@ func (s *DockerSuite) TestVolumeCliCreate(c *check.C) {
 	c.Assert(name, check.Equals, "test")
 }
 
+func (s *DockerSuite) TestVolumeCliCreateOptionConflict(c *check.C) {
+	dockerCmd(c, "volume", "create", "--name=test")
+	_, _, err := dockerCmdWithError("volume", "create", "--name", "test", "--driver", "nosuchdriver")
+	c.Assert(err, check.NotNil, check.Commentf("volume create excepted to be failed on creating an existed volume with a different driver name"))
```

Comment:
can we test that the error we get is the right error we expect? I'm not completely sure this command fails where we expect it to fail.